nselib/http.lua

```diff
@@ -1252,6 +1252,45 @@ local function request(host, port, data, options)
   return response
 end
 
+--- This function merges the cookies received in <code>response.cookies</code> 
+-- to the cookies that already exist in the options. 
+--
+-- The merge is based on RFC 6265 and when a different cookie with same <code>
+-- name</code>, <code>path</code> and <code>domain</code> is received, it replaces
+-- the old cookie, else it gets appended at the end of <code>options.cookies</code table.
+
+-- @param response The response received from the server
+-- @param options The options table having previously received cookies.
+-- @return A response table
+-- @return An options table, with the cookies from the response table appended.
+-- @see get
+local function merge_cookie_table(response, options)
```

Comment:
This function is still not taking in considerations cookie attributes like Expires or Max age.